.github/workflows/dockershim.yml

```diff
@@ -0,0 +1,123 @@
+name: critest dockershim
+on: [push, pull_request]
+jobs:
+  #
+  # Run CRI tests against dockershim
+  #
+  build-and-critest-dockershim:
+    strategy:
+      matrix:
+        os: [ubuntu-18.04] # Need to fix windows-2019 critest
+                           # It was also disabled in travis ci
+    name: ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Install Go 1.14
+        uses: actions/setup-go@v1
+        with:
+          go-version: '1.14'
+
+      - name: Set Go binary path
+        shell: bash
+        run: |
+          echo "::set-env name=GOPATH::${{ github.workspace }}"
+          echo "::add-path::${{ github.workspace }}/bin"
+
+      - name: Checkout cri-tools for this commit
+        uses: actions/checkout@v2
+        with:
+          path: ${{github.workspace}}/src/github.com/kubernetes-sigs/cri-tools
+
+      - name: Install docker
```

Comment:
may already be installed pls check

Reply:
Not sure this is an issue at moment. It install into the workspace path and then copied to `/usr/local/bin`. Have a look at `hack/install-cri-dockerd.sh` and let me know what you think.

Reply:
I mean I think docker is installed with the platform as one of the defaults.. I'm curious the version of docker that comes with github actions works for this docker shim.

Reply:
if we just install.. is there a particular version we want... who sets that.. k8s, or dockershim?

Reply:
Following discussion with @mikebrow, installing particular version in `install-diocker.sh`.